lib/css/components/_stacks-popovers.less

```diff
@@ -0,0 +1,236 @@
+//
+//  STACK OVERFLOW
+//  POPOVERS
+//
+//  This CSS comes from Stacks, our CSS & Pattern library for rapidly building
+//  Stack Overflow. For documentation of all these classes and how to contribute,
+//  visit https://stackoverflow.design/
+//
+//  TABLE OF CONTENTS
+//  • VARIABLES
+//  • BASE STYLE
+//  • ARROWS
+//  • LOCATIONS
+//
+//  ============================================================================
+//  $   BASE STYLES
+//  ----------------------------------------------------------------------------
+.s-popover {
+    visibility: hidden;
+    position: relative;
+    min-width: 12rem;
+    max-width: 24rem;
+    width: 100%;
+    padding: @su12;
+    z-index: @zi-hide;
+    opacity: 0;
+    border-radius: @br-md;
+    border: 1px solid @black-100;
+    background-color: @white;
+    box-shadow: @bs-sm;
+    color: @fc-dark;
+    font-size: @fs-body1;
+
+    will-change: visibility, z-index, opacity; // Not supported in IE11 / Edge
```

Comment:
Is `will-change` solving an actual problem here or is it just there preemptively? If the latter, I would prefer removing it. To quote [MDN](https://developer.mozilla.org/de/docs/Web/CSS/will-change#Summary),

> *Don't apply will-change to elements to perform premature optimization.* If your page is performing well, don't add the `will-change` property to elements just to wring out a little more speed.